cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -623,46 +653,6 @@ SwiftGenerator::writeServantDocSummary(IceInternal::Output& out, const Interface
     }
 }
 
-void
-SwiftGenerator::writeMemberDoc(IceInternal::Output& out, const DataMemberPtr& p)
```

Comment:
Apart from the order that we generated things in, this was identical to `writeDocSummary`, and only called in one place.
I deleted this one, and now we just call `writeDocSummary` for everything.